Detections/MultipleDataSources/PhosphorusIOCs.yaml

```diff
@@ -1,32 +1,38 @@
 ﻿id: 155f40c6-610d-497d-85fc-3cf06ec13256
-name: example showing how to query for domain IOCs across multiple data sources
+name: Known Phosphorus group domains
 description: |
-  This query looks for exact match domain name IOCs related to Phosphorus group discussed 
-  in https://blogs.microsoft.com/on-the-issues/2019/03/27/new-steps-to-protect-customers-from-hacking/
+  'Matches domain name IOCs related to Phosphorus group activity with CommonSecurityLog, DnsEvents and VMConnection dataTypes.
+  References: https://blogs.microsoft.com/on-the-issues/2019/03/27/new-steps-to-protect-customers-from-hacking/.'
 severity: High
 requiredDataConnectors:
   - connectorId: DNS
     dataTypes:
       - DnsEvents
-  - dataTypes:
-     - VMConnection
-     - CommonSecurityLog
-  - connectorId: Syslog
+  - connectorId: VMConnection
```

Comment:
There is no connectoId "".  
The connectorId is Optional field. not all dataTypes have connectors.

you can see the connectorId map in file:
https://microsoft.sharepoint.com/:x:/t/AzureSecurityInsights/ESUNgcoX6shFpq_QJM11Rq4BuOFq05c2s7fShsNjncXqVA?rtime=sOOPomcp10g